data/i18n/hi/hi.toml

```
@@ -240,6 +247,21 @@ other = "(रिलीज़ड: "
 [release_date_format]
 other = "2006-01-02"
 
+# Deprecated. Planned for removal in a future release.
+# Use [release_full_details_initial_text] instead.
+[release_complete]
+other = "पूरा"
+
+# Replace [release_complete] with [release_full_details_initial_text]
+[release_full_details_initial_text]
+other = "पूरा"
+
+[release_schedule]
+other = "अनुसूची"
+
+[release_changelog]
+other = "बदलाव का"
```

I think that `अभिलेख` here closely matches with log (its actual meaning is record). Also we could transliterate changelog to `चेंजलॉग` too(if it is more easily associable than its localization). Thoughts, WDYT @NitishKumar06 ?
```suggestion
other = "बदलाव अभिलेख"
```

---

This sounds good to me. Also, Do you think "and" in line no. 09 should be converted to Hindi too?

---

Yes that `and` on line number 9 should be localized too. (Sorry for missing out on that.)